xblockutils/public/studio_container.js

```diff
@@ -0,0 +1,55 @@
+function StudioContainerXBlockWithNestedXBlocksMixin(runtime, element) {
+    var $buttons = $(".add-xblock-component-button", element);
+
+    function isSingleInstance(button) {
+        return $(button).data('single-instance');
+    }
+
+    $buttons.click(function(ev) {
+        if ($(this).is('.disabled')) {
+            ev.preventDefault();
+            ev.stopPropagation();
+        } else {
+            if (isSingleInstance(this)) {
+                $(this).addClass('disabled');
+                $(this).attr('disabled', 'disabled');
+            }
+        }
+    });
+
+    function updateButtons() {
+        var nestedBlockLocations = $.map($(".studio-xblock-wrapper"), function(block_wrapper) {
```

Comment:
Should `$(".studio-xblock-wrapper")` be grabbing from the global space, or selecting from a subsection of `element`?